Functions/xASL_quant_GetControlLabelOrder.m

```diff
@@ -16,11 +15,12 @@
 % DESCRIPTION: This function automatically checks (and corrects if required)
 %              the control and label order of ASL timeseries
 %              based on the larger signal in control volumes.
-%              It supposes that data is acquired in pairs.
+%              It supposes that data is acquired in pairs. Works also for multiPLD but only for sequences
+%              with alternative control/label or label/control order
 %
 % EXAMPLE:
-% [~, ~, OrderContLabl] = xASL_quant_GetControlLabelOrder(ASLTimeSeries, x);
-% [ControlIm, LabelIm] = xASL_quant_GetControlLabelOrder(ASLTimeSeries, x);
+% [~, ~, OrderContLabl] = xASL_quant_GetControlLabelOrder(ASLTimeSeries);
+% [ControlIm, LabelIm] = xASL_quant_GetControlLabelOrder(ASLTimeSeries);
 % 
 % __________________________________
 % Copyright (C) 2015-2019 ExploreASL

```

Comment:
Maybe update the date while we're at it, otherwise perfect 👍

Reply:
Done!